components/builder-worker/habitat/config/config.toml

```diff
@@ -3,6 +3,8 @@ auto_publish = {{cfg.auto_publish}}
 data_path = "{{pkg.svc_data_path}}"
 bldr_channel = "{{cfg.bldr_channel}}"
 bldr_url = "{{bind.depot.first.cfg.url}}"
+github_app_id = 5565
```

Comment:
We'll want to make this configurable since we'll have an app_id for acceptance, too